### ISSUE TITLE
export multiple screens as a gif anim

### ISSUE THREAD

**Issue by nurpax** [nurpax/petmate#44]
TODO:

* [x] Need fps control in export options dialog
* [ ] Optional but nice to have: progress bar in the export dialog (some 30-40 frame export can take roughly 1s)

forked from https://github.com/nurpax/petmate/issues/19

**Comment by nurpax** [comment]
Might want to have screen reordering for this one too: https://github.com/nurpax/petmate/issues/85

**Comment by nurpax** [comment]
![anim](https://user-images.githubusercontent.com/297823/44752445-a5488b00-ab23-11e8-9126-2a90e2a0d682.gif)

**Comment by nurpax** [comment]
I implemented this but it is SUPER slow.  I profiled this in Chrome devtools and turns out it's spending most of the time recomputing a new quantized palette for the input frames in the gif-encoder's neuquant (that legendary Kosonen SOM neural net).  This is of course quite wasteful since we don't need to compute a palette as the inputs can only ever contain 16 palette entries.  

I'm planning to patch gif-encoder by adding an option to pass in a palette that's used for quantization. This should make gif export run several orders of magnitude faster.
